samples/wifi/shell/overlay-zperf.conf

```diff
@@ -7,7 +7,7 @@ CONFIG_NET_PKT_RX_COUNT=30
 CONFIG_NET_PKT_TX_COUNT=30
 CONFIG_NET_BUF_RX_COUNT=30
 CONFIG_NET_BUF_TX_COUNT=60
-CONFIG_HEAP_MEM_POOL_SIZE=230000
+CONFIG_HEAP_MEM_POOL_SIZE=140000
```

Comment:
Shouldn't this now be zero? IIRC, this requirement is purely from Wi-Fi driver and now that we are using dedicated pools, this shouldn't be used. And instead we need to use the newly introduced defines here.

Reply:
Have reduced the `CONFIG_HEAP_MEM_POOL_SIZE` after introduction of dedicated heap pools for driver. This is to accommodate other subsystems which use `k_malloc`.